modules/stats/Classes/Stats.php

```diff
@@ -62,19 +68,19 @@ public function __construct()
 
             // Update search engine data
             $search_engine = '';
```

Comment:
This is a block that appears to be in need for a check if this is still needed.
#AOLSerarchEngineNo1

Reply:
I guess same applies for altavista ;)
This is just a piece of older code that need to be revised.
But it is kind of working right now.